test_scripts/RC/CLIMATE_RADIO/GetInteriorVehicleData/014_Absence_of_subscribe_parameter_in_request_for_HMI_in_case_of_2nd_Subscription_UnSubscription.lua

```diff
@@ -75,23 +75,28 @@ runner.Title("Test")
 for _, mod in pairs(commonRC.modules) do
   -- app has not subscribed yet
   runner.Step("Unsubscribe app to " .. mod, subscriptionToModule, { mod, false })
-  runner.Step("Send notification OnInteriorVehicleData " .. mod .. ". App is not subscribed", commonRC.isUnsubscribed, { mod })
+  runner.Step("Send notification OnInteriorVehicleData " .. mod .. ". App is not subscribed",
```

Comment:
@VjKlepikov As I can see there was just step function moved to next line here and further. What for this change?

Reply:
@aderiabin The step function moved to the next line due to the length over 180 characters